stock_orderpoint_generator/models/orderpoint_template.py

```diff
@@ -36,12 +36,8 @@ class OrderpointTemplate(models.Model):
         help="Auto compute minimum quantity "
              "per product for a given a date range",
     )
-    auto_min_date_start = fields.Datetime(
-        string="From",
-    )
-    auto_min_date_end = fields.Datetime(
-        string="To",
-    )
+    auto_min_date_start = fields.Datetime()
```

Comment:
@chienandalu Why removing string ?

Reply:
He has removed them in favor of the automatic ones for avoiding the warning of "duplicated labels", which in fact is more correct. They are put however on the view instead, for better display.

Reply:
Ok, saw in form, but for exports or ... the name will be Auto Min Date Start which could lead to inconsistencies

Reply:
Well, on export you don't have context, so better to have full name, although it's not the same as you see in screen.

Reply:
Yes, that was the exact reason :)